docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-responsive-design.md

```diff
@@ -172,9 +172,31 @@ All design tokens are configurable per screen size, which allows you to control
 
 The layout system is covered in more detail in the layout documentation.
 
-## Responsive Visibility Rules
+## Responsive Images
+
+Images are shown in various places throughout the experience. For example, small icon-sized images appear in a typeahead search results and increase in size when they're shown in carousel or grid, and might blow up when you see them in a detail page. Some images require fullscreen size and others are only rendered inside a small layout.
+
+The web platform has several ways to optimized image per viewport. Oryx components, such as the `ProductMediaComponent` and `ImageComponent` design system component are fully equipped to render optimized images for the right screen size and even device pixel density and current network connection speed.
+
+It is however important to have the right images available. This can be done automatically which is offered by 3rd party service.
```

Comment:
Sounds too abstract. What is this 3rd party service? I would make some accent that this is regarding any image optimisers, and `Cloudinary` as an example
```
It is however important to have the right images available. This can be done automatically which are offered by 3rd party services, for example Cloudinary.
```

Reply:
did a tiny rephrase, but also there's already the example of cloudinary in the info box below this sentence. Have you seen that?